.github/workflows/ci.yml

```diff
@@ -10,8 +10,8 @@ on:
 
 env:
   # Common versions
-  GO_VERSION: '1.19.5'
-  GOLANGCI_VERSION: 'v1.50.1'
+  GO_VERSION: '1.20.x'
```

Comment:
Is there a feature that we want to get with this new version? Otherwise, I am wondering if we should wait a bit to get  it stabilized for a while?

Reply:
Also, having a fixed version would produce more consistent builds, i.e. `1.20.1` instead of `1.20.x`

Reply:
> Is there a feature that we want to get with this new version? 

Nope.

> Otherwise, I am wondering if we should wait a bit to get it stabilized for a while?

Sure

Reply:
@turkenh fixed.